templates/omero-web-config-signup.j2

```diff
@@ -10,4 +10,20 @@ config set -- omero.web.signup.group.name %Y-%m
 config set -- omero.web.signup.group.templatetime true
 config set -- omero.web.signup.email.enabled true
 config set -- omero.web.signup.helpmessage 'If you have trouble creating an account please contact {{ omero_server_mail_from | default('help@example.org') }}'
-config set -- omero.web.signup.email.body 'Your login details for {{ inventory_hostname }} are:\n\nusername: {username}\npassword: {password}\n'
+config set -- >-
```

Comment:
Sorry, I made a mistake, this is a Jinja2 file so YAML syntax doesn't work. What would work is putting the variable in the playbook yml file, and using a variable here. E.g.
```suggestion
config set -- omero.web.signup.email.body '{{ omero_signup_email_body }}'
```
and in the playbook vars set
```yml
omero_signup_email_body: >-
  Thank you for creating an account on demo.openmicroscopy.org server.
  Your login details are \n\n
  username: {username}\n
  ...
```